x-pack/plugins/ml/public/application/services/ml_api_service/inference_models.ts

```diff
@@ -18,17 +19,19 @@ export function inferenceModelsApiProvider(httpService: HttpService) {
      * @param taskType - Inference Task type. Either sparse_embedding or text_embedding
      * @param modelConfig - Model configuration based on service type
      */
-    createInferenceEndpoint(
+    async createInferenceEndpoint(
       inferenceId: string,
       taskType: InferenceTaskType,
       modelConfig: ModelConfig
     ) {
-      return httpService.http<estypes.InferencePutModelResponse>({
+      const result = await httpService.http<estypes.InferencePutModelResponse>({
         path: `${ML_INTERNAL_BASE_PATH}/_inference/${taskType}/${inferenceId}`,
         method: 'PUT',
         body: JSON.stringify(modelConfig),
         version: '1',
       });
+      await savedObjectsApiProvider(httpService).syncSavedObjects(false);
```

Comment:
As you're using the ML Kibana API, could you please kick off SO sync server-side [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/ml/server/routes/inference_models.ts#L48)?

Reply:
Updated in https://github.com/elastic/kibana/pull/187154/commits/c013a7629675ffb52bfebb085e3e15a90c48b5e8

Reply:
fixed in https://github.com/elastic/kibana/pull/187154/commits/9ab82c5de0a94968c508f318f22703d373f7ff64